### PR TITLE
Missing `uglifier` dependency in Gemfile

### PR DESCRIPTION
**Pull request by bishboria**
Deploy on preview (#1453) failed due to uglifier not being found.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ gem 'govuk_admin_template', '1.4.2'
 
 gem 'airbrake', '3.1.15'
 gem 'plek', '1.7.0'
-gem 'json', '1.7.7'
+gem 'json', '1.8.0'
```
Comment (bradleywright) [inline Gemfile]: Seems unrelated to change?

File: Gemfile [hunk 2]
```diff
@@ -41,6 +41,10 @@ gem 'rake', '10.4.1'
 
 gem 'sass-rails', '4.0.3'
 
+group :production do
```
Comment (bradleywright) [inline Gemfile]: This should be in `assets`: the command failing is:

    rake#RAILS_ENV=production RAILS_GROUPS=assets assets:precompile

Comment (bishboria) [inline Gemfile]: I spoke to @jennyd and she recommended removing the group I added…

Which is the preference?

Comment (bradleywright) [inline Gemfile]: Having just read the Rails 4 upgrade guide @jennyd is correct.

File: Gemfile (hunk 1 above)
Comment (bishboria) [inline Gemfile]: Uglifier needs that version of json
